bin/people_array.rb

```diff
@@ -4,3 +4,33 @@
 people = People.new.people
```

Comment:
I believe the `bin.people_array.rb` file is used for the lab where developers annotate the code.  
@jrhorn424 can you confirm whether this file should be deleted or not.
https://github.com/ga-wdi-boston/ruby-enumerable-io/tree/016/master#challenge---annotate-unfamiliar-code